challenges/platform/devops.md

```diff
@@ -0,0 +1,122 @@
+# Summary
```

Comment:
The challenge with levels looks good to me - I like the combination of troubleshooting + scripting, a good mix.